setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Oh my bad, I tried to navigate to the root which gave me a 404. Apologies for that. I don't know how up to date it is actually. It's a while since I did a release :disappointed: .